docs/_documentations/troubleshooting.md

```diff
@@ -75,6 +75,27 @@ When using OS authentication setups (for example, AzureAD), Docker Shared Drive
 2. Create the new account with the same username but without the prefix (for example, if your AzureAD account is `AzureAD/BobSmith`, your new local account should be `BobSmith`). Use the same password as your other account.
 3. Select your new local account and click **Change account type**. Select the dropdown menu and select **Administrator**. Share the drive again in Docker.
 
+<!--
+Action/Topic: Installing Codewind
+Issue type: bug/info
+Issue link: https://github.com/eclipse/codewind/issues/436
```

Comment:
Shouldn't we link to a user issue, not the doc issue? like https://github.com/eclipse/codewind/issues/411 ?

Reply:
This is actually metadata and not viewable from the landing pages. Also, 436 links to 411, so I think this is OK as-is for our purposes here. :) Thank you for catching this, though!